src/index.js

```diff
@@ -2,17 +2,20 @@
 
 const koa = require('koa')
 const koaRouter = require('koa-router')
+const fs = require('fs')
 
 const {registerMediator, activateHeartbeat} = require('openhim-mediator-utils')
 
 const logger = require('./logger')
-const mediatorConfig = require('../mediatorConfig.json')
 const config = require('./config')
 
+const mediatorConfigFile = fs.readFileSync(__dirname +'/../mediatorConfig.json')
```

Comment:
@MattyJ007 Let's try to use the `path` module here instead of string concatenation as explained in [this](https://github.com/standard/standard/issues/403) issue